scripts/travis_script.sh

```diff
@@ -41,7 +41,8 @@ if [[ $GROUP == docs ]]; then
 
     # Verify tutorial docs build
     pushd docs
-    pip install "sphinx<1.8" sphinx_rtd_theme recommonmark
```

Comment:
I wonder why we needed this old version...

Reply:
Sphinx 1.8 broke our sphinx build. I think it has been patched in the interim, and I needed this for `-W --keep-going` to convert warnings into errors for CI.